lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
@matthew-andrews Oh sorry I read that line out of context. I did a similar fix [here](https://github.com/Financial-Times/origami-build-tools/pull/332/files). I'm just trying to work out what the difference is between yours and mine... Mine doesn't try to establish a 'root', and just resolves every request to where it really is. But yours may be more resiliant in some other way I don't understand :) can you check and say which you think is better